db/seeds.rb

```diff
@@ -5,3 +5,7 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+survey1 = Survey.create(survey_name: "U.S. Household Food Security Survey", survey_category: "Food Security", languages: "English, Spanish, Chinese")
```

Comment:
languages is currently one string, not a list, should probably be updated to something this, or whatever ruby's syntax is for arrays: ["English", "Spanish", "Chinese"]